exec/bytecode/struct_test.go

```diff
@@ -41,9 +41,11 @@ func TestStruct(t *testing.T) {
 	expect(t,
 		func() {
 			b.DefineVar(v)
-			b.Push("Name")
+			b.Push(0)
+			b.MakeArray(reflect.TypeOf([]int{0}), 1)
 			b.Push("bar")
-			b.Push("Age")
+			b.Push(1)
+			b.MakeArray(reflect.TypeOf([]int{1}), 1)
 			b.Push(30)
 			b.Struct(reflect.TypeOf(p), 2)
```

Comment:
```go
foo := Foo{
   A: 1,
   B: "Hi",
   C: Bar{D: 1},
}
```

And we can't use:

```go
foo := Foo{
    A: 1,
    B: "Hi",
    C.D: 1,
}
```

So, we don't need a `[]int` to indicate index, `int` is enough.

Reply:
Oh! Very good example, thanks~!